symfony/webpack-encore-bundle/1.0/package.json

```diff
@@ -8,9 +8,9 @@
     "license": "UNLICENSED",
     "private": true,
     "scripts": {
-        "dev-server": "encore dev-server",
-        "dev": "encore dev",
-        "watch": "encore dev --watch",
-        "build": "encore production --progress"
+        "dev-server": "NODE_ENV=development encore dev-server",
+        "dev": "NODE_ENV=development encore dev",
+        "watch": "NODE_ENV=development encore dev --watch",
+        "build": "NODE_ENV=production encore production --progress"
```

Comment:
For portability reasons something like [`cross-env`](https://github.com/kentcdodds/cross-env#readme) should also probably be used.

Reply:
Yes, if we do this, then we need to use `cross-env`. I was always hoping to avoid that. 

An alternative might be documentation? Couldn't we advise people to set this env var (of some other env var) inside `webpack.config.js` if they want to? Like:

```
// webpack.config.js
process.env. NODE_ENV = Encore.isProduction() ? 'production' : 'development';
```

WDYT?

Reply:
I made some changes according to your comments. What do you think? Thanks.